src/plugins/lowMach.cpp

```diff
@@ -54,8 +55,16 @@ void lowMach::setup(nrs_t* nrs, dfloat gamma)
   platform->options.setArgs("LOWMACH", "TRUE"); 
 }
 
-// qtl = 1/(rho*cp*T) * (div[k*grad[T] ] + qvol)
 void lowMach::qThermalIdealGasSingleComponent(dfloat time, occa::memory o_div)
+{
+  nrs_t* nrs = the_nrs;
+  cds_t* cds = nrs->cds;
+  expansionCoeff = 1; 
```

Comment:
This need to be removed. Instead set `expansionCoeff=1` in qThermalRealGasSingleComponent.